R/utils.R

```diff
@@ -564,3 +553,7 @@ validate_epi_data <- function(epi_data) {
   }
   return(epi_data)
 }
+
+get_bucket_df_delphi <- function(prefix = "", bucket = "forecasting-team-data") {
+  aws.s3::get_bucket_df(prefix = prefix, bucket = bucket)
+}
```

Comment:
This is unused?

Reply:
Yea, it's purely a convenience wrapper for me messing with bucket files in the REPL.